### PR TITLE
done with remove reception

### PR DESCRIPTION
**Pull request by DoctorFreed**

### CODE REVIEW

File: src/modules/controllers/reception.controller.js [hunk 1]
```diff
@@ -72,3 +72,22 @@ module.exports.editRecept = async (req, res) => {
...
+  const { id } = req.person;
+
+  if (id_post == null || !id_post)
+    return res.status(422).send("Field id_post cannot be empty");
+
+  const reception = await Reception.findOne({ where: { id: id_post } });
+  if (reception.dataValues.user_id != id)
+    return res.status(403).send("This is not your reception!");
+
+  if (Reception.findOne({ where: { id: id_post } }) == null)
```
Comment (usatovaDasha) [inline src/modules/controllers/reception.controller.js:87]: you have reception higher

File: src/modules/controllers/reception.controller.js [hunk 2]
```diff
@@ -72,3 +72,22 @@ module.exports.editRecept = async (req, res) => {
...
+    return res.status(404).send("This reception does not exist!");
+
+  Reception.destroy({ where: { id: id_post } })
+    .then((result) => res.json(result))
```
Comment (usatovaDasha) [inline src/modules/controllers/reception.controller.js:91]: what will the system information about the deletion give you? and how will it help the user who made the request?

Comment (DoctorFreed) [reply src/modules/controllers/reception.controller.js:91]: he return count deleting post, it turns out he always sent 1, client can process 1 how true

Comment (DoctorFreed) [reply src/modules/controllers/reception.controller.js:91]: i already have 404 check, yep this info useless, removed this

File: src/modules/controllers/reception.controller.js (hunk 1 above)
Comment (DoctorFreed) [reply src/modules/controllers/reception.controller.js:87]: fixed

**Pull request merged**